components/Chatbar/ChatbarSettings.tsx

```diff
@@ -30,7 +30,7 @@ export const ChatbarSettings: FC<Props> = ({
 }) => {
   const { t } = useTranslation('sidebar');
   return (
-    <div className="flex flex-col items-center space-y-1 border-t border-white/20 pt-1 text-sm">
+    <div className="flex flex-col w-full space-y-1 border-t border-white/20 pt-1 text-sm">
```

Comment:
button style used below for ok/cancel requires a `w-full` and omitting `items-center`